osmaxx-py/osmaxx/tests/test_utils.py

```diff
@@ -0,0 +1,31 @@
+import string
+import random
+from django.test import TestCase
+import os
+from osmaxx.utils import PrivateSystemStorage
+
+
+class PrivateSystemStorageTestCase(TestCase):
+    def setUp(self):
+        random_directory_name_length = 10
+        self.random_directory_name = ''.join(
+            random.choice(string.ascii_lowercase + string.digits) for _ in range(random_directory_name_length)
```

Comment:
It signifies that this is a throwaway variable, it isn't used at all, just to be thrown away. It is quite common for Python to do it this way.